app/helpers/lightweight_activity_helper.rb

```diff
@@ -67,4 +67,19 @@ def activity_player_url(activity)
   def activity_player_page_url(activity, page)
     return  "#{activity_player_url(activity)}&page=#{page.position}"
   end
+
+  def activity_player_conversion_url(activity)
+    if @sequence
+      params[:lara_resource] = "#{api_v1_activity_url(@sequence.id)}.json"
+      params[:resource_name] = @sequence.title
+    else
+      params[:lara_resource] = "#{url_encode(api_v1_activity_url(activity.id))}.json"
+      params[:resource_name] = activity.name
+    end
+
+    url = ENV['CONVERSION_SCRIPT_URL']
+    uri = URI.parse url
+    uri.query = URI.encode_www_form URI.decode_www_form(uri.query || "").concat(params.to_a)
+    return uri.to_s
+  end
```

Comment:
@scytacki Is this pretty much what you intended for adding the params to the conversion script URL?

Reply:
The way this is done mostly in the portal is like this:
https://github.com/concord-consortium/rigse/blob/60fdf4f894316382af1ab16925c3ef0eb1385ca1/rails/app/controllers/external_activities_controller.rb#L57-L62

That seems like the best way to go.